vasyl_varyshchuk/Week_1/task_3/generate_file_size.py

```diff
@@ -0,0 +1,53 @@
+"""
+This module returns random size of file
+"""
+
+import random
+
+list_of_files = ['reports.txt', 'monthly_reports.zip', 'README.txt',
+                 'picture.jpg', 'logo.ttf', 'config.ini']
+
+
+def recognize_file_type(file_name):
+    _, extension = file_name.split('.')
+    return extension
+
+
+def txt_handler():
+    return random.randint(100, 450)
+
+
+def zip_handler():
+    return random.randint(500, 700)
+
+
+def jpg_handler():
+    return random.randint(800, 1000)
+
+
+def ttf_handler():
+    return random.randint(1100, 1300)
+
+
+def ini_handler():
+        return random.randint(1300, 1500)
+
+
+functions = {
```

Comment:
It's better to call it `handlers` instead.

Reply:
Thanks for the advice. I renamed it as you recommended.